crossmint-client-sdk.code-workspace

```diff
@@ -23,6 +23,10 @@
         {
             "name": "🏁 Starters",
             "path": "packages/starter"
+        },
+        {
+            "name": "✅ @crossmint/verify-credentials-sdk",
+            "path": "packages/credentials/verify"
```

Comment:
i dont think this path really makes sens

i would just make it `packages/verifiable-credentials`

Reply:
I did that to be consistent with other packages

Reply:
Changed